debian/control

```diff
@@ -67,7 +67,7 @@ Build-Depends: debhelper-compat (= 13),
                python3-pyelftools [amd64 i386 arm64 armhf riscv64] <!stage1>,
                python3-pyparsing <!nocheck>,
                python3-evdev:native <!nocheck>,
-               tzdata-legacy <!nocheck> | tzdata (<< 2023c-8) <!nocheck>,
+               tzdata-legacy <!nocheck> | tzdata <!nocheck>,
```

Comment:
How odd. I think it would be better to cherry pick https://salsa.debian.org/systemd-team/systemd/-/commit/e9b57c6d847fe202ee02b3758b1b71276a180b99. I think the point of all of this is that the tests depend on some timezones in tzdata-legacy, which is a new split out package in Debian testing containing deprecated timezones. The version constraint is there [to ensure tzdata-legacy is considered](https://salsa.debian.org/systemd-team/systemd/-/commit/9c63b8d33f2aa42bd890a8bd3322ab23fcca84d5).

Reply:
Good suggestion!

Reply:
I backport [Update tzdata build dependency constraint to 2024a-1](https://salsa.debian.org/systemd-team/systemd/-/commit/e9b57c6d847fe202ee02b3758b1b71276a180b99) to replace `debian/control: Fix the build dependecy of tzdata`